cloudproxy/check.py

```diff
@@ -44,7 +44,13 @@ def fetch_ip(ip_address):
 
 def check_alive(ip_address):
     try:
-        result = requests.get("http://" + ip_address + ":8899", timeout=3)
+        proxies = {
+            "http": "http://" + ip_address + ":8899",
+            "https": "http://" + ip_address + ":8899"
+        }
+        result = requests.get("http://google.es", timeout=3, proxies=proxies)
```

Comment:
Risk of spamming Google with requests and Google may decide to block the IP, which would impact folks using this to scrape off Google.

I agree with the change but better to hit https://api.ipify.org instead as they expect this behavior.